src/sonarwhal-theme/layout/partials/scripts.hbs

```diff
@@ -34,6 +36,7 @@
         });
     }());
 </script>
-<script src="{{url_for theme.offline.registerWorkerPath}}"></script>
```

Comment:
There are a few remaining references to `swRegisterTemplate.js` that still need to be updated (run: `npm run server`).